### ISSUE TITLE
napi.h obsoleted by NeXus -> write "pure" hdf5-based solution for McCode

### ISSUE THREAD

**Issue by willend** [McStasMcXtrace/McCode#531]
It turns out

**Comment by farhi** [comment]
The napi is now included in our distribution

**Comment by willend** [comment]
This is true, but only because it is still included in the NeXus (GitHub) sources. I think a "pure hdf5" solution must be done eventually (and is on an ESS "roadmap" in fact).

**Comment by willend** [comment]
Update 2026: Increasingly important to eventually replace...
